README.md

```diff
@@ -1,2 +1,127 @@
 # MihertabDamena.github.io
-My first web
+capstone project - ICT EXPO Conference page
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖MY Portfolio -ICT EXPO Conference page: <a name="about-project"></a>
+
+>ICT EXPO Conference page project is created using HTML and CSS Use JavaScript to manipulate DOM elements. Here on the project, I prepared Two page for both mobile and desktop users .
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- BUILT WITH -->
+
+## 🛠 Built With <a name="built-with"></a>
+
+- Programming languages:   HTML, CSS3
+- Tools:  Git, GitHub, Linters
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+- Responsive
+- Mobile first
+- Popup Mobile and Desktop window
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Link to deployed project.
+
+- [Live Demo Link](https://damenam.github.io/ICT-EXPO-ETHIOPIA-Conference)
+- [videos Link](https://www.loom.com/share/04eed985fdfd4a2a83b10d548dd91ef0))
```

Comment:
- [ ] As the previous reviewer suggested, kindly make sure to include a piece of code you are proud of making so you can showcase your ability to describe your code for none technical person